NOTICE

```diff
@@ -3,3 +3,7 @@ Copyright [2014-2016] The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+
+Portions of this software were developed at
```

Comment:
http://www.apache.org/legal/src-headers.html#notice

in special:

```...
The NOTICE file may also include copyright notices moved from source files submitted to the ASF.
```


I found at least one example doing this.. I have seen others but I have the link for this one now:

https://github.com/apache/incubator-mynewt-core/blob/master/NOTICE